index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var semver = require('semver');
+var execa = require('execa');
+var del = require('del');
+
+var exec = function (cmd, args) {
+	const result = execa.sync(cmd, args);
+
+	if (result.stderr !== '') {
+		throw new Error(result.stderr);
+	}
+
+	return result.stdout;
+};
+
+module.exports = function (input) {
+	input = input || 'patch';
+
+	if (['patch', 'minor', 'major'].indexOf(input) === -1 && !semver.valid(input)) {
+		throw new Error('Invalid version.');
```

Comment:
Error message can be better. Might be something like

- `Version should be one of patch, minor, major or a valid semver version`